src/api.tsx

```diff
@@ -2,8 +2,44 @@ import { apiSecret } from "./models";
 
 const apiKey = import.meta.env.VITE_API_KEY;
 
-export const getRestaurantList = async (range: number) => {
-  const uri = `/api/search?sort_by=best_match&limit=${range}&location=Milano`;
+export const getRestaurantList = async ({
+  prices,
+  location,
+  radius,
+}: {
+  prices: boolean[];
+  location: string;
+  radius: number;
+}) => {
+  const priceParamsString: string = prices
+    .map((price, index) => {
+      if (price) {
+        return `price=${index + 1}`;
+      }
+    })
+    .filter((price) => price !== "")
```

Comment:
If you have not defined a return for the `price == false` case, are you sure the filter is working correctly?
```suggestion
    .map((price, index) => price ? `price=${index + 1}` : "")
    .filter((price) => price !== "")
```